TG.Backend/Features/Blob/CreateBlobs/CreateBlobsCommandHandler.cs

```diff
@@ -0,0 +1,49 @@
+using TG.Backend.Models.Blob;
+using TG.Backend.Repositories.Blob;
+using TG.Backend.Services;
+
+namespace TG.Backend.Features.Blob.CreateBlobs;
+
+public class CreateBlobsCommandHandler : IRequestHandler<CreateBlobsCommand, BlobResponse>
+{
+    private readonly IBlobRepository _blobRepository;
+    private readonly IAzureBlobStorageService _azureBlobStorageService;
+
+    public CreateBlobsCommandHandler(IBlobRepository blobRepository, IAzureBlobStorageService azureBlobStorageService)
+    {
+        _blobRepository = blobRepository;
+        _azureBlobStorageService = azureBlobStorageService;
+    }
+
+    public async Task<BlobResponse> Handle(CreateBlobsCommand request, CancellationToken cancellationToken)
+    {
+        foreach (var blobFile in request.Blobs)
+        {
+            var blob = new Data.Blob
+            {
+                Name = blobFile.FileName
+            };
+
+            var blobId = await _blobRepository.CreateBlobAsync(blob);
+            await _azureBlobStorageService.UploadBlobAsync(blobId, blobFile);
+        }
+        
+        // próbowałem w ten sposób ale nie idzie bo ef ma problem z parallel wątkami, moze ty masz pomysl lepszy
```

Comment:
That's ok, but I think that we can use await foreach here (if I am not mistaken, however that foreach loop isn't bad)
https://learn.microsoft.com/en-us/archive/msdn-magazine/2019/november/csharp-iterating-with-async-enumerables-in-csharp-8#a-tour-through-async-enumerables

Reply:
This is not possible in this case, because IFormFileCollection does not implement IAsyncEnumerable<T>